fusion-cli/entries/client-entry.js

```diff
@@ -16,6 +16,9 @@ function reload() {
   const main = require('__FUSION_ENTRY_PATH__'); // eslint-disable-line
   const initialize = main.default || main;
   Promise.resolve(initialize()).then(app => {
+    if (!app) {
+      throw new Error('Application entry point did not return an App');
```

Comment:
should this check be done inside the `initialize()` function to avoid repeating?

Reply:
Unfortunately we can't. `initialize` is the default export from `src/main.js` in the consumer application. This check was just to make sure that function always returns a valid application